index.pug

```diff
@@ -0,0 +1,96 @@
+html
+  head
+    link(rel="stylesheet" href="/style.css")
+    script(src="../cars.js")
+    body
+     h1.🚦MY CAR COLLECTION API 🏎
+
+     h2. here's a curated list of cars desired by customers /people
+     
+    ul.list 
+      p Get the cars listed 
+
+      li.carList
+        p 
+          strong GET 
+          | /allCars
+        div#getAllCars
+            p Method: GET 
+            p Route: http://localhost:2035/cars/allCars   
+            p Desired outcome : this should bring out all the cars 
+
+      li.listItems 
+            p 
+          strong GET 
+          |/all Cars By Id
+        div#getById
+            p method : GET 
+            p route: http.//localhost:7825/cars/allCars{id}
+            p these are all the cars by their id 
+        p 
+          strong POST 
+          | /allCars
+        div 
+            p Method: POST 
+            p Route: http://localhost2035/cars/allCars
+            p Headers: content type : application/JSON
+            p Body: {model, type, year, modelname, founder}
+            p Desired Outcome ; cerates new fileds with list of new cars. 
+
+            li.listCars
+        p 
+          strong DELETE 
+          | /car By Id
+        div 
+            p Method: DELETE 
+            p Route: http://localhost2035/cars/CarsById
```

Comment:
You'd need a way to provide the id of the car you'd be deleting. Something like `DELETE /cars/1`